src/zkevm_specs/evm/execution/error_invalid_opcode.py

```diff
@@ -0,0 +1,38 @@
+from ...util import FQ
+from ..execution_state import ExecutionState
+from ..instruction import Instruction, Transition
+from ..opcode import invalid_opcodes
+from ..table import CallContextFieldTag
+
+
+def invalid_opcode(instruction: Instruction):
+    # Fixed lookup for invalid opcode.
+    opcode = instruction.opcode_lookup(True)
+    instruction.responsible_opcode_lookup(opcode)
+
+    # Constrain it is an invalid opcode.
+    instruction.constrain_in(opcode, [FQ(i) for i in invalid_opcodes()])
```

Comment:
I could not confirm if need to constrain it is an invalid opcode here.

Reply:
in the soec we can check like this, think about how to check in real rust circuit.

Reply:
Need to update this code according to circuit.